Automattic/wp-calypso
# Add support for a `/<locale>/plans` route

Related to https://github.com/Automattic/i18n-issues/issues/641

## Proposed Changes

In email and support documents, we often link to
wordpress.com/plans. Logged in users are directed
to the plans available for their domain in Calypso, 
but logged out users are redirected to an English
pricing page (wordpress.com/pricing)

This commit adds support for a plans page that includes the locale. 
We'll update the `localizeUrl` function to add the locale to this
page and redirect the user accordingly


## Testing Instructions



* Visit the `/plans` and `/es/plans` URL on the `calypso.live` URL, and verify that it continues to go to `wordpress.com/plans/<site>` for logged in users
* Visit `/es/plans` and verify that it redirects to the localized pricing page for logged-out users

## Pre-merge Checklist
